.github/workflows/test.yml

```diff
@@ -26,3 +26,8 @@ jobs:
           cache: 'maven'
       - name: Run the Maven verify phase
         run: mvn --batch-mode --update-snapshots verify
+
+        env:
+          JOB_TYPE: test
+          SPANNER_EMULATOR_HOST: localhost:9010
+          GOOGLE_CLOUD_PROJECT: spark-spanner-test
```

Comment:
Is this actual project ? Maybe we can pass the project id from system environment variables, or so.

Reply:
So I am trying to have it in the Cloud Spanner emulator so its value won't matter at all, until we hit actual full integration tests, which are currently running locally.